### DISCUSSION TITLE
campaign randomevents feedback part 2 electric boogaloo

### DISCUSSION THREAD

**Discussion by LePiromano** [FakeFishGames/Barotrauma#8655] [Unstable feedback]
here i once again talk about how monsters spawn through the campaign and the general experience of the campaign overall
i'd also like to remind you this is from a very (insanely) experienced player but i try to see the game from a new-ish player viewpoint, and i also gather some info from actual new players too

The main things that changed are the new monsters and I'm also gonna talk about the new progressively difficult abyss monsters.
Cold Caverns (5 to 20% difficulty)
The first 3 to 5 levels are quite boring for anyone who isn't playing for the first time ever unless they really like running into ice spires. You can get very easily breached by a few crawlers (mostly after fixing a beacon unless you have bots on the guns or a paranoid captain) or a single mudraptor if you are standing still, but enemies that can actually heavily damage the submarine while it's moving don't exist yet. As always you can get to endgame gear in this area if you really want to, since alien ruins and wrecks spawn even in the lowest of difficulties, meaning there's little to no defense inside of these (unless you're playing with jovian rad on). You'll meet spinelings in the cold caverns but unless you REALLY suck at aiming coilguns they're not a threat at all. There's also the very small chance of finding a random hammerhead but since it's only one it'll just make a big hole and then die the next time it tries to hit you.

I'd say this area is fine over all, I just wouldn't spawn any structure of loot unless you get a mission that includes those.

Europan Ridge (21 to 35% difficulty)
Mostly the same, the main problem is that small enemies seem to just not be as deadly if the sub is moving horizontally for whatever reason. The main difference is the increased hammerhead chance and the new abyss creature spawning, tho at these difficulties it's not really worth going abyss mining yet. By the time you're in the europan ridge you probably have at least some upgrades in the sub, which makes getting sunk by small creatures even more unlikely. You've also probably got a box of depleted fuel ammo in this biome which is insane, but that'll be fixed in the talent re-balance (...right?). The enviromental hazards that are the floating icebergs and currents prevents you from speedrunning through levels sometimes which is not something I dislike. You'll probably meet your first naturally spawned molochs in this biome, which can be a big deal for those who have no idea that they're almost entirely blind. There's also the crawler broodmother which unless you run into every single one of it's eggs will probably not sink you since the chance of its vomit projectiles hitting the same spots twice is low and each projectile by itself is not enough to cause a problematic leak. Then there's the threshers which aren't a big problem for now since if the player is smart they'll kill the bonethresher really fast, and probably won't let the tiger threshers break the doors.

If you did what I advised on the cold caverns, then you should introduce the wrecks in this biome but not the alien ruins yet. Also a few mudraptor veterans and a rare giant spineling would make sense.

Aphotic Plateau (36 to 50% difficulty)
Here the spawns get a bit interesting, unfortunately there's no enviromental hazards other than ballast flora, which would be a problem if it wasn't on such small packs, it'd be really cool if the aphotic plateu's patches of ballast flora spores were much larger than those in other biomes, to follow the fungal theme of the biome. Here thresher swarms can get a bit heated if you don't kill the bonethreshers before they latch to your hull and frickin wreck it making a gigantic hole for tiger thresher to go through, which is awesome. The golden hammerheads do feel quite different from normal hammerheads and can wreck you if you are not careful.
This is the biome where abyss mining starts becoming viable, you just gotta pray you get the new abyss creature instead of the charybdis. The watcher also starts appearing here but... it doesn't do much. The endworm starts spawning but the chance is so low you probably won't see it in this biome, but when it does it gives you a heart attack.

If you've been following with the suggestions, this is the biome where I'd start introducing the alien ruins, with like a 50 or 60% chance of spawning each level. Also, I think the watcher needs some kind of change, it always runs away at the slightest touch leaving all of its buffed creatures powerless, or simply can't keep up, so you'll never notice that it even buffs creatures playing normally. Maybe massively increase the radius of the buff, so it doesn't come near the sub until all of its allies are dead and make the gazer rage effect scale up faster, I dunno.

The Great Sea (51 to 65% difficulty)
Here's where everything starts going batshit and you can't get sunk otherwise you get overwhelmed, I personally like this but I'm not sure if it might get tedious for new players. My only problem here is those frickin annoying hammerhead swarms that never kill any of your crew but always take your sub down, other than that most of the monsters in the great sea are a considerable but not annoyingly huge threat, the giant spineling would be the exception here if it didn't run away after every hit like the watcher and the golden hammerhead do. Abyss mining here starts being quite rewarding but the increasing chance of getting an endworm gets me (and probably anyone) paranoid. The black moloch can absolutely wreck your shit and put your sub down for good if you don't kill it before it blows up your supercapacitors and you don't have the handheld weapons to take it down by hand.

Here's where there'd be a 100% chance of alien ruins spawning of course, other than that I have no suggestions for the biome or the random events other than making hammerhead do something more original than just ramming until they die. The matriarch just makes a big hole and puts some annoyances that you can get rid of with a welder inside of your sub, if they even manage to get in.

Hydrothermal Wastes (66 to 80% difficulty)
Every single damn level in this biome feels like 100% hellish back in 0.10, 3 molochs and a black one just beating the shit out of the sub while being attacked by like 8 hammerheads and you're fighting off a thalamus at the same time. You have absolutely no choice other than speeding through the levels hoping that you trigger no random events other than the distance-based ones, or going to the abyss, since at this point you're better off hoping you get a charybdis attacking you than staying inside the actual level.

I honestly don't know what could you even do to make this biome challenging without just being hell (even tho it looks like it), unless you want to go back to the randomevents from a few updates ago.

??? (100%)
hell yeah worm-based submarine propulsion

Abyss monsters feedback:
Right now it comparing the three abyss creatures is like comparing a swarmfeeder and hammerhead spawn to a black moloch.
The new one just dies from any kind of weapon for now since it has no armor anywhere other than its legs. Its tongue attack doesn't do much but if it bites you or shoots the spikes at you it'll shed the hull like paper, mostly because you'll meet it when you don't yet have the full hull upgrades yet.
The charybdis requires a lot of skill to land shots in its mouth, mostly if using a railgun, but you can also brute force it with the other 3 kind of weapons with a lot of ammo, it can also sink you considerably fast.
Then the endworm can just get shot by 50 physicorium shells and only get a few armor pieces torn off, the new chances of limb severing are a bit ridiculous and the only limb you can consistently sever is the furthest from the head. I think you should consider a more skill based system for the endworm's limb system, maybe giving each limb its own health.
(shameless plug to my own suggestion https://github.com/Regalis11/Barotrauma/discussions/8654)

over all I think the random events have improved a lot and barotrauma's finally earning that hardcore reward it won back in 2018
once again feel free to add other feedback and other suggestions in this thing's comments

**Comment by itchyOwl** [comment]
Once again thanks for the feedback. Notes taken!

**Comment by cshingle** [comment]
I like your suggestions. I may need to create an event mod based on this.

I have done quite a few runs now and the part of the game that I find most boring is the Hydrothermal Wastes. There are way less stations and therefor less missions to do. Killing abyss monster missions are fun, but after a few you just start avoiding them because they are just a resource sink. I run public games and this part is when the crew has the least to do, and the captain has the most. This is bad because a crew that is not killing monsters will kill each other.

I modified the campaign save file to increase the difficulty a while back. We reached %100 difficulty when we reached the Hydrothermal Wastes and the boss worm spawned on every level. That complicated things a bit because we had to slip past him each level until the last where we got revenge. It was the best time I have had in the Hydrothermal Wastes.

**Reply by LePiromano** [reply]
oh yeah I remember when the abyss creatures actually came up to the level to chase you down when you had a hunting grounds side quest because there was nothing stopping them from just leaving the abyss, absolutely one of the most fun things that we had in this game

honestly chase sequences have huge potential in this game
